module-system/sov-modules-api/src/default_context.rs

```diff
@@ -31,6 +31,8 @@ impl Spec for DefaultContext {
 
 #[cfg(feature = "native")]
 impl Context for DefaultContext {
+    type GasUnit = TupleGasUnit<1>;
```

Comment:
It should be `TupleGasUnit<2>` the same as for the `ZkContext` it is because we need to charge for both executions.

Reply:
Makes sense, fixed!